File: Beecrowd/1002 Área do Círculo/main.cpp
#include <iostream>
#include <math.h> 
using namespace std;
 
int main() {
 
	double raio, n = 3.14159, area;
	
	cin >> raio;
	
	area = n * pow(raio, 2);
	cout << fixed;
	cout.precision(4);
	cout << "A=" << area << endl;
 
    return 0;
}

File: Beecrowd/1002 Área do Círculo/Main.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);
        
        double area, raio;
        final double N = 3.14159;

        raio = sc.nextDouble();

        area = N * Math.pow(raio, 2.0);

        System.out.printf("A=%.4f%n", area);

        sc.close();
    }
 
}